integration/js/steps/StartMeetingReadinessCheckerStep.js

```diff
@@ -20,12 +20,9 @@ class StartMeetingReadinessCheckerStep extends AppTestStep {
   }
 
   async run() {
-    const sleep = (milliseconds) => {
-      return new Promise(resolve => setTimeout(resolve, milliseconds))
-    };
-
+    await TestUtils.waitAround(10000);
```

Comment:
Do we need it anymore? Since we already wait for the Button to be enabled?

Reply:
Removed.